From 8f1b3e4da6e030ac04319a44dcdcc5e0d09d7ecd Mon Sep 17 00:00:00 2001
From: Lance <Gero3977@gmail.com>
Date: Fri, 22 Apr 2022 04:22:34 -0700
Subject: [PATCH] hwdb: Fix rotation for HP Pro Tablet 408 G1

(cherry picked from commit 6dc18ca5ddd996ffbf83b42cb45f23496fe38c9d)

Related: #2087778
---
 hwdb.d/60-sensor.hwdb | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/hwdb.d/60-sensor.hwdb b/hwdb.d/60-sensor.hwdb
index b54539bf53..d6a584150c 100644
--- a/hwdb.d/60-sensor.hwdb
+++ b/hwdb.d/60-sensor.hwdb
@@ -453,6 +453,10 @@ sensor:modalias:acpi:SMO8500*:dmi:*:svnHewlett-Packard:pnHPStream8Tablet:*
 sensor:modalias:i2c:bmc150_accel:dmi:*:svnHewlett-Packard:pnHPPavilionx2Detachable:*:rn815D:*
  ACCEL_MOUNT_MATRIX=0, -1, 0; -1, 0, 0; 0, 0, 1
 
+# HP Pro Tablet 408 G1
+sensor:modalias:i2c:bmc150_accel:dmi:*:svnHewlett-Packard:pnHPProTablet408:*:rn8048:*
+ ACCEL_MOUNT_MATRIX=0, 1, 0; -1, 0, 0; 0, 0, 1
+
 #########################################
 # I.T.Works
 #########################################
